docs/API/components/list-from-data-source.md

```diff
@@ -224,7 +229,10 @@ Fliplet.App.Storage.set('flDynamicListQuery:simple-list', {
 ```js
 Fliplet.App.Storage.set('flDynamicListQuery:simple-list', {
   search: { ... },
-  previousScreen: true,
+  goBack: {
+    enableButton: true,
+    action: `(function () { console.log('run code') })`
```

Comment:
avoid using templates strings in the examples since they are not supported on IE